packages/terra-action-header/src/ActionHeaderContainer.scss

```diff
@@ -0,0 +1,60 @@
+:local {
+  .flex-header {
+    background-color: var(--terra-action-header-background-color, #f4f4f4);
+    border-bottom: var(--terra-action-header-border-bottom, 1px solid #dedfe0);
+    border-left: var(--terra-action-header-border-left);
+    border-right: var(--terra-action-header-border-right);
+    border-top: var(--terra-action-header-border-top);
+    box-shadow: var(--terra-action-header-box-shadow);
+    display: flex;
+    justify-content: space-between;
+    padding: var(--terra-action-header-padding, 0.714rem);
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-fill {
+    align-self: center;
+    flex: 1 1 auto;
+    height: 100%;
+    position: relative;
+    width: 100%;
+  }
+
+  .flex-end {
+    align-items: center;
+    display: flex;
+    flex: 0 0 auto;
```

Comment:
I think it was actually the .flex-collapse rule that needed updating. I also added brackets when using multiple classes with cx [here](https://github.com/cerner/terra-core/commit/5183093e2ead29ad95062eb5251e6dd714f5e6a9) since this could cause issues with older browsers.